lib/stub_ui/lib/painting.dart

```diff
@@ -1023,6 +1023,11 @@ enum Clip {
 // constant and can't propagate into the set/get calls.
 const Endian _kFakeHostEndian = Endian.little;
 
+/// Indicates to the engine to not resize this dimension.
+///
+/// This needs to be kept in sync with "kDoNotResizeDimension" in codec.cc
```

Comment:
that comment doesn't make sense to expose to our users.

this documentation should mention where this is used.

so e.g. something like:

```dart
/// Indicates that the image should not be resized in this dimensions.
///
/// Used by [instantiateImageCodec] as a magical value to disable resizing in the given direction.
//
// This needs to be kept in sync with "kDoNotResizeDimension" in codec.cc
```

(notice the switch to `//` near the end)